### DISCUSSION TITLE
Downmix from > 2 Channels to 2 Channels (Stereo)

### DISCUSSION THREAD

**Discussion by flowcool** [ptr727/PlexCleaner#255] [Q&A]
Hello Everyone,

A quick question of configuration... In another forum ([unmanic](https://github.com/Unmanic/unmanic/issues/376)), I saw a quite good idea but don't know how to implement it if feasible...

I have a basic TV and stereo would be more than fine; and would like to downmix > 2 channels to stereo.

Trick is there:
`-af pan=stereo|c0=c2+0.30*c0+0.30*c4|c1=c2+0.30*c1+0.30*c5
`
But wonder if I can put that straight away in the config file ?

Cheers,

**Comment by eric10k93** [comment]
did you get this working?  I'm looking to potentially use this software to auto-downmix audio... mainly, I can't play files that only have 7.1 audio, so was hoping to downmix to 5.1 or stereo

**Reply by ptr727** [reply]
Not supported, and no intention of adding this as there is no logic to inspect audio channel layouts, and this is what Plex/Emby/Jellyfin is supposed to do based on the specific players' playback capabilities, i.e. downmix/encode if the player can't support the stream.

There are other tools that can do bulk re-encoding, or you can do it by batch file, keep in mind the complexity comes in analyzing the different audio streams, then analyzing the track layout, and doing a correct downmix mapping of the right tracks, e.g. like `ffmpeg -ac 2` does.
